camayoc/qcs_models.py

```diff
@@ -295,7 +316,19 @@ def equivalent(self, other):
                 'Sources objects or dictionaries.'
             )
 
-        for key, value in self.fields().items():
+        local_items = self.fields()
+        local_keys = local_items.keys()
+        other_keys = other.keys()
+        all_keys = set().union(*[local_keys, other_keys])
+        for key in all_keys:
```

Comment:
updated this to merge the sets of keys so we make sure that not only does the server have everything we have locally, it also has set any extra data to something sane.

Reply:
The only concern I have about this is that it may bypass any field that should not have being returned or is extra and we should update camayoc. I rather prefer to know when something new is returned or something is not returned anymore than relying on the testing system to return it to me.